CMakeSettings.json

```diff
@@ -93,6 +93,21 @@
       "addressSanitizerRuntimeFlags": "detect_leaks=0",
       "variables": []
     },
+    {
+      "name": "WSL-Clang-Release-ASAN",
+      "generator": "Ninja",
+      "configurationType": "Release",
+      "buildRoot": "${projectDir}\\out\\build\\${name}",
```

Comment:
can we unify build & install root dir for x64/86 and WSL?

Reply:
where should be that unified location?

Reply:
the one for the x64/86 was OK IMO.